lib/widgets/dialog.dart

```diff
@@ -46,17 +58,18 @@ DialogStatus showErrorDialog({
   final zulipLocalizations = ZulipLocalizations.of(context);
   final future = showDialog<void>(
     context: context,
-    builder: (BuildContext context) => AlertDialog(
+    builder: (BuildContext context) => AlertDialog.adaptive(
       title: Text(title),
       content: message != null ? SingleChildScrollView(child: Text(message)) : null,
       actions: [
         if (learnMoreButtonUrl != null)
-          TextButton(
+          _adaptiveAction(
             onPressed: () => PlatformActions.launchUrl(context, learnMoreButtonUrl),
-            child: _dialogActionText(zulipLocalizations.errorDialogLearnMore)),
-        TextButton(
+            text: zulipLocalizations.errorDialogLearnMore,
+          ),
```

Comment:
nit: preserve formatting
```suggestion
            text: zulipLocalizations.errorDialogLearnMore),
```